### PR TITLE
completed CSS positioning

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-12-03 at 10 32 01 AM
Screen Shot 2021-12-03 at 11 17 23 AM
Screen Shot 2021-12-02 at 3 35 44 PM

### CODE REVIEW

**Comment by dcatan89** [comment]
Kapture 2021-12-03 at 11 26 19
excuse the last picture

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  right: 0;
+  right: 15px;
+  top: 20px;
+  z-index: 7;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css]: This is unnecessary 🤔

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  right: 0;
+  right: 15px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css]: You should never declare the same property twice in the same rule. Keep just one of these, and delete the other.

File: css-positioning/absolute.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom: 0;
+  z-index: 6;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css]: This is unnecessary 🤔

File: css-positioning/absolute.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
+  position: absolute;
+  z-index: 5;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css]: This is unnecessary 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by dcatan89** [comment]
Z-indexes removed and also the right property under emoji class

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**